cypress/support/commands.js

```diff
@@ -321,7 +321,7 @@ Cypress.Commands.add('click_filter_button', () => {
 	cy.get('.filter-selector > .btn').click();
 });
 
-Cypress.Commands.add('click_listview_primary_button', (btn_name) => {
+Cypress.Commands.add('click_primary_button', (btn_name) => {
 	cy.get('.primary-action').contains(btn_name).click({force: true});
 });
```

Comment:
Other apps are using [this command](https://github.com/erpnext/erpnext_ui_tests/blob/2e2695833e0ee834615d978bffd6f36de46c61e9/cypress/integration/test_customer.js#L9) as well... better to create a new one?